tests/sentry/integrations/msteams/test_message_builder.py

```diff
@@ -36,6 +45,35 @@ class MSTeamsMessageBuilderTest(TestCase):
     These tests do NOT test all visual aspects of the card.
     """
 
+    def setUp(self):
+        self.user = self.create_user(is_superuser=False)
+        owner = self.create_user()
+        self.org = self.create_organization(owner=owner)
+
+        self.integration = Integration.objects.create(
+            provider="msteams",
+            name="Fellowship of the Ring",
+            external_id="f3ll0wsh1p",
+            metadata={
+                "service_url": "https://smba.trafficmanager.net/amer",
+                "access_token": "y0u_5h4ll_n07_p455",
+                "expires_at": int(time.time()) + 86400,
+            },
```

Comment:
What's this metadata for?

Reply:
It's not necessary for this test I guess, let me see if I can get rid of it without making the integration object unhappy.

Reply:
Removed.